acceptance/cmd/fs/cp/dir-to-dir/script

```diff
@@ -0,0 +1,6 @@
+mkdir -p localdir
```

Comment:
Instead of creating these files at test runtime, you can create and commit them here. That way, the test can directly read them without having to write them first.

I recommend commiting input files directly unless the file contents are dynamic.